### PR TITLE
Updated documented view summary table

### PR DESCRIPTION
**Pull request by ellenitoumpas**

### CODE REVIEW

File: MATH2269_final_project.Rmd [hunk 1]
```diff
@@ -672,102 +697,175 @@ if(hasnt_run(trial_type)){
...
 
-```{r Posterior Images, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "", fig.width = 10}
+#### model_002_gamma_gamma_c3_a500_b1000_t5
+
+```{r}
+
+trial_type <- "model_002_gamma_gamma_c3_a500_b1000_t5"
+
+# Parameters to use
+use_parameters <- list(number_chains = 3,
```
Comment (tgrrr) [inline MATH2269_final_project.Rmd:729]: This has been removed in https://github.com/ellenitoumpas/bayesian-final-project/pull/23, which I've merged, so you'll likely get conflicts anywhere that use_parameters is specified.

File: MATH2269_final_project.Rmd [hunk 2]
```diff
@@ -672,102 +697,175 @@ if(hasnt_run(trial_type)){
...
+
+```{r Regression Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "The regression parameter model diagnostics for trial model_002_gamma_gamma_c3_a500_b1000_t5.", fig.width=10}
+
+view_regression_diagnostics_images(trial_type)
+
+```
+
+```{r Prediction Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "A sample of the prediction model diagnostics for trial model_002_gamma_gamma_c3_a500_b1000_t5.", fig.width = 10}
+
+view_prediction_diagnostics_images(trial_type)
```
Comment (tgrrr) [inline MATH2269_final_project.Rmd:769]: Also repeated

File: MATH2269_final_project.Rmd [hunk 3]
```diff
@@ -672,102 +697,175 @@ if(hasnt_run(trial_type)){
...
+  
+}
+
+```
+
+This model took `r view_trial_info(trial_type)$duration %>% round(1)` to run.
+
+```{r Regression Diagnostics, echo=FALSE, fig.show = "hold", out.width = "50%", fig.align = "default", fig.cap= "The regression parameter model diagnostics for trial model_002_gamma_gamma_c3_a500_b1000_t5.", fig.width=10}
+
+view_regression_diagnostics_images(trial_type)
```
Comment (tgrrr) [inline MATH2269_final_project.Rmd:763]: This looks like it's repeated below here: https://github.com/ellenitoumpas/bayesian-final-project/pull/26/files#diff-71d7f9db18c1cf529943173af95c9441R702

**Review by tgrrr — COMMENTED** [review]
This looks pretty good. 

Except that I think we might get some conflicts. And some functions seem to be run multiple times.

For the conflicts:
This PR contains many similar changes to https://github.com/ellenitoumpas/bayesian-final-project/pull/25.
We'll get far fewer conflicts with smaller, compartmentalised PR's, branched from `master`

You might be able to minimise git conflicts by:
1. merge `update_document_view_summary_table` into `update_initial_values_functions`
2. Try to merge that branch into `deleted_subsample_function`
3. merge `master` into `deleted_subsample_function` (this isn't a typo)
4. If that all works, and there's no conflicts, and the Rscript runs: merge `deleted_subsample_function` into `master`

pay special attention to conflicts with `use_parameters`

File: MATH2269_final_project.Rmd (hunk 3 above)
Comment (ellenitoumpas) [reply MATH2269_final_project.Rmd:763]: As I have mentioned we need to do it like this. This function will output a set of diagnostic we will then need to write comments on in the report side of things. There is no way we can condense this/ functionise it anymore.

File: MATH2269_final_project.Rmd (hunk 2 above)
Comment (ellenitoumpas) [reply MATH2269_final_project.Rmd:769]: On purpose. Repeated for the second model. It is a requirement to do so!

**Pull request merged**